src/utils/googleAnalytics/initialize.ts

```diff
@@ -6,8 +6,9 @@ declare global {
   }
 }
 
-export const gtag = function (...rest: unknown[]) {
-  window.dataLayer.push(...rest);
+export const gtag = function (..._arg: unknown[]) {
```

Comment:
```suggestion
// eslint-disable-next-line @typescript-eslint/no-unused-vars
export const gtag = function (..._arg: unknown[]) {
```

여기도 lint ignore 부탁드려요

Reply:
네! 반영했습니다.